### ISSUE TITLE
Illegal character in path error

### ISSUE THREAD

**Issue by nikolauska** [headswe/Swifty#108]
#### Swifty version
2.1.1.2

#### Description 
RHS version 0.4 throws illegal character in path error

#### Error message
```
2015-12-17 22:20:57,645 [4] INFO SwiftUpdater.Pages.Download - Starting download for addons/rhs_t72/data/anim/MG_CommanderOut ? kopia.rtm_0.spd
2015-12-17 22:21:04,708 [4] INFO SwiftUpdater.Pages.Download - Crash
System.ArgumentException: Illegal characters in path.
   at System.IO.Path.CheckInvalidPathChars(String path, Boolean checkAdditional)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   at SwiftPbo.PboArchive.ExtractAll(String outpath) in h:\Development\C#\SwiftPbo\SwiftPbo\PboArchive.cs:line 290
   at SwiftUpdater.Pages.Download.ExtractPbos(List`1 changedPbos) in h:\Development\C#\Arma\AddonLoader\SwiftUpdater\SwiftUpdater\Pages\Download.cs:line 184
   at SwiftUpdater.Pages.Download.ExtractPbos() in h:\Development\C#\Arma\AddonLoader\SwiftUpdater\SwiftUpdater\Pages\Download.cs:line 165
   at SwiftUpdater.Pages.Download.Work() in h:\Development\C#\Arma\AddonLoader\SwiftUpdater\SwiftUpdater\Pages\Download.cs:line 294
   at SwiftUpdater.Pages.Download.Worker_DoWork(Object sender, DoWorkEventArgs e) in h:\Development\C#\Arma\AddonLoader\SwiftUpdater\SwiftUpdater\Pages\Download.cs:line 319
```

**Comment by firefly2442** [comment]
Can also confirm the error.

**Comment by headswe** [comment]
Error is confirmed, awaiting time to fix, might try to do a emergency release on the 25th to disable PBO patching

**Comment by ryanparsons7** [comment]
Temporary fix: Delete rhs_t72.pbo and redownload the entire pbo

**Comment by headswe** [comment]
Aye, almost ready with a fix.

**Comment by headswe** [comment]
Should be fixed now with 2.1.1.3
